common/djangoapps/track/backends/oli.py

```diff
@@ -0,0 +1,121 @@
+"""OLI analytics service event tracker backend."""
+
+from __future__ import absolute_import
+
+import json
+import logging
+
+from requests_oauthlib import OAuth1Session
+
+from django.contrib.auth.models import User
+
+from student.models import anonymous_id_for_user
+from track.backends import BaseBackend
+
+
+LOG = logging.getLogger(__name__)
+
+
+class OLIAnalyticsBackend(BaseBackend):
+    """
+    Transmit events to the OLI analytics service
+    """
+    def __init__(self, **kwargs):
+        super(OLIAnalyticsBackend, self).__init__(**kwargs)
+
+        # the oli analytics service endpoint will be at
+        # "{}{}".format(oli_analytics_service_url, path)
```

Comment:
This comment should be implementation-agnostic, not referencing
`str.format` as that's a design choice. The issue with referencing
design choices is that they easily fall out-of-sync. In fact, that
appears to have already happened here; the comment says "str.format" but
the code actually says "str + str". The comment also still references
the older, more verbose variable name.

At the least, this should say something like "The endpoint is generated
by concatenating the url and path". But at that point, I'm not sure that
really tells us anything important (endpoints typically are created by
appending a path to a base URL).

I would probably just remove this comment entirely. That said, if you
feel it adds needed clarity to the code, I think that moving it to the
docstring (of either the method or the class) would be preferred over an
inline comment, as it explains how the parameters are used.